### PR TITLE
Improving Region Availability

### PR DESCRIPTION
**Pull request by marvinbuss**
## Summary of the Pull Request
This PR improves region availability and adds few new regions as outlined in the changed files. This PR also fixes a few bicep linting rules that are shown.

## References
This will fix: https://github.com/Azure/data-management-zone/issues/256

## PR Checklist
* [x] Closes Issue https://github.com/Azure/data-management-zone/issues/256
* [x] Code of Conduct signed. If not, go to [Code of Conduct](https://github.com/Azure/data-management-zone/blob/main/CODE_OF_CONDUCT.md).

## Validation Steps Performed
Please see checks below. Also, users reported successful deployments.

### CODE REVIEW

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Review by marvinbuss — COMMENTED** [review]
LGTM!

File: .github/linters/.arm-ttk.psd1 [hunk 1]
```diff
@@ -4,11 +4,13 @@
 @{
     # Test = @( )
     Skip = @(
-        'Template Should Not Contain Blanks',
-        'DeploymentTemplate Must Not Contain Hardcoded Uri',
-        'DependsOn Best Practices',
-        'Outputs Must Not Contain Secrets',
-        'IDs Should Be Derived From ResourceIDs',
+        'Template Should Not Contain Blanks'
```
Comment (amanjeetsingh) [inline .github/linters/.arm-ttk.psd1:7]: Please replace "should" with "must". 

Thanks

**Review by amanjeetsingh — COMMENTED** [review]
One minor change proposed. 
Other than that, good to go!

File: .github/linters/.arm-ttk.psd1 (hunk 1 above)
Comment (marvinbuss) [reply .github/linters/.arm-ttk.psd1:7]: This is a predefined rule. We cannot change this or functionality of the linter would change.

**Review by amanjeetsingh — APPROVED** [review]

**Pull request merged**